flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/runtime/utils/JavaUserDefinedScalarFunctions.java

```diff
@@ -318,4 +320,91 @@ public PythonEnv getPythonEnv() {
             return null;
         }
     }
+
+    /** Increment input. */
+    public static class AsyncJavaFunc0 extends AsyncScalarFunction {
```

Comment:
move those functions into `AsyncCalcTestPrograms` to have complete test programs. functions should not be shared across tests.